tests/unit/helpers/url-helper-test.php

```diff
@@ -106,6 +106,23 @@ public function test_get_extension_from_url() {
 		$this->assertEquals( $expected, $actual );
 	}
 
+	/**
+	 * Tests retrieving of the file extension.
+	 *
+	 * @covers ::get_extension_from_url
+	 * @covers ::get_url_path
+	 */
+	public function test_get_extension_from_url_null() {
+		Monkey\Functions\expect( 'wp_parse_url' )
+			->with( 'https://example.com', \PHP_URL_PATH )
+			->andReturn( null );
+
+		$expected = '';
+		$actual   = $this->instance->get_extension_from_url( 'https://example.com' );
+
+		$this->assertEquals( $expected, $actual );
+	}
```

Comment:
* Why are you testing this via the `get_extension_from_url()` method ?
* Does this actually test what should be tested ? Or rather, maybe the question should be: what is it that _should_ be tested/what is the functionality you are trying to safeguard ?
* Why are you mocking the return value of `wp_parse_url()` ? Could the function be stubbed to return the return value of the PHP native `parse_url()` instead ?